### ISSUE TITLE
[Question] Indexing in/out edges in OrientDB 3.0.x

### ISSUE THREAD

**Issue by andreyvk** [orientechnologies/orientdb#8765]
### OrientDB Version: 3.0

Hi,

Is it still possible to index in/out attributes in edges?

I have created an `in` `LINK` attribute on a specific edge that we have. However, querying that class of edges did not seem to gain me any speed. Is there anything that I am doing wrong here?

```
create index has_a.in on has_a (in) notunique
select from has_a where in = #125:1443671
```

**Comment by luigidellaquila** [comment]
Hi @andreyvk 

I'd say it should work fine, it's just a normal property (if it doesn't, it's definitely a bug).
I'm adding it to my backlog, I'll try to look into it soon

Thanks

Luigi

**Comment by luigidellaquila** [comment]
in the meantime, could you please run an EXPLAIN of the query?

Thanks

Luigi

**Comment by andreyvk** [comment]
hi @luigidellaquila ,

```
+ FETCH FROM CLASS has_a
  + FETCH FROM CLUSTER 130 ASC
  + FETCH NEW RECORDS FROM CURRENT TRANSACTION SCOPE (if any)
+ FILTER ITEMS WHERE 
  in = #125:1443671
```

**Comment by luigidellaquila** [comment]
Oh yes, it's not using the index...

**Comment by luigidellaquila** [comment]
Hi @andreyvk 

I found and fixed the problem, the fix will be released with next v 3.0.16

Thanks

Luigi
